flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -311,7 +304,7 @@ class FlxTilemapExt extends FlxTilemap
 	
 	/**
 	 * THIS IS A COPY FROM FlxTilemap
-	 * I've only swapped lines 386 and 387 to give DrawTilemap() a chance to set the buffer dirty
```

Comment:
Actually I didn't dig in this method, but I think is a @mrcdk modification. Comparing to original `FlxTilemap` seems a little outdated. I don't know how `FlxTileSpecial` works neither how to use it, may be good someone else check it.